example/shared/css/index.css

```diff
@@ -171,4 +171,11 @@ ul li:nth-last-child(1) {
     #your-products {
         margin-top: 1em;
     }
+    footer {
+        margin-top: 1em;
+        padding-top: 1em;
+        border-width: 1px 0 0 0;
```

Comment:
how do I do that? My approach to CSS is hit everything with a mallet until it stops screaming.